lib/test_cases/regex/99.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Use entity repository and not hastranslation",
+  "description": "Use EntityRepository::getTranslationFromContext instead of hasTranslation.",
+  "links": [
+    {
+      "title": "Deals with the translation of values.",
```

Comment:
Title is the reference link title in the popup shown, so it should appropriately describe the link:
Maybe something like this:
Entity Translation EntityRepository::getTranslationFromContext

Reply:
Acknowledged